data/adopters.yaml

```diff
@@ -5,12 +5,15 @@ adopters:
   - name: "Department of Defense"
     link: https://www.defense.gov/
     logo: https://www.access-board.gov/images/dod-seal.png
+  - name: "Die Mobiliar"
+    link: https://www.mobiliar.ch/
+    logo: https://mobiliar.rokka.io/dynamic/crop-width-701-height-525-mode-ratio--resize-width-701--options-autoformat-true-optim.quality-4/dfeb319d1c438de49d67799cf87534f3f332f498/logo-der-mobiliar-.jpg
   - name: EPAM Delivery Platform
     link: https://epam.github.io/edp-install/
     logo: https://raw.githubusercontent.com/epam/edp-install/master/docs/assets/edp-logo-150x150-black.png
   - name: Fastweb
     link: https://www.fastweb.it/
-    logo: https://www.fastweb.it/grandi-aziende/gfx/common/logo-fastweb-header.svg
+    logo: https://www.fastweb.it/adsl-fibra-ottica/gfx/sprite.svg?rel=9c0c9d68#v8662a44f
```

Comment:
Maybe this one is better: https://upload.wikimedia.org/wikipedia/commons/thumb/a/a0/Fastweb_logo.svg/2560px-Fastweb_logo.svg.png